.gitignore

```diff
@@ -0,0 +1,5 @@
+/bin/
+/build/
+/lua/
+/luajit/
+/scons/scons-local-2.2.0/
```

Comment:
@rbehrends: there are still some files in `/scons/scons-local-2.2.0/` which are under version control, could you check that this is fine, please?